smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaSource.java

```diff
@@ -98,7 +98,7 @@ public KafkaSource(Vertx vertx, Config config, String servers) {
 
     public void closeQuietly() {
         try {
-            this.consumer.closeAndAwait();
+            this.consumer.closeAndForget();
```

Comment:
With the priority thingy this should not be required anymore.

Reply:
Do you mean revert to the original code or remove?

Reply:
yes. closeAndAwait();

Reply:
Should be the right code now